### ISSUE TITLE
Two Radio Bugs.

### ISSUE THREAD

**Issue by unit0016** [tgstation/tgstation#63547]
Reporting client version: 514.1568

## Round ID:
[175194](https://scrubby.melonmesa.com/round/175194)

## Testmerges:
- [Revert "Require Node 16"](https://www.github.com/tgstation/tgstation/pull/63331)
- [test disables ai static to see if it helps maptick enough to justify a refactor of how it works](https://www.github.com/tgstation/tgstation/pull/63439)

## Reproduction:
Short and sweet.
Intercomms are playing radio despite their speaker being turned off (This occured on the round ID In question), and The Orion Trail is spouting both security and medical comms around it. (I originally thought it was a positronic brain, thus the stupid chain of edits this thread's already received from me. At least it's clarified now.)

I heavily suspect these two are related, and are being grouped as such.

**Comment by SuperNovaa41** [comment]
@Kylerace

**Comment by Mooshimi** [comment]
>acting as a weird, pseudo-syndiecomms key by saying this do you mean the posi brain recieved every radio channel?

**Comment by unit0016** [comment]
> >acting as a weird, pseudo-syndiecomms key
> 
> by saying this do you mean the posi brain recieved every radio channel?

Yeah. (Minus syndicate itself, mind)
I was on charlie station at the time, and the fact I was hearing medical comms was writing me-

WAIT

I.
May have done a brain hurty.

It was probably the Orion Trail, actually. I built a cabinet there and started receiving medical comms after, hence the second one. Apologies for the mixup, editing the initial to acknowledge it.

I'll do an in-game test to confirm in a few minutes, I just woke up.

**Comment by Mooshimi** [comment]
Some related lines in orion.dm where radio is mentioned:
https://github.com/tgstation/tgstation/blob/3418cd7aaf852faed9240169e009e68b1eeaff58/code/game/machinery/computer/arcade/orion.dm#L41-L49
https://github.com/tgstation/tgstation/blob/3418cd7aaf852faed9240169e009e68b1eeaff58/code/game/machinery/computer/arcade/orion.dm#L127-L135

**Comment by Kylerace** [comment]
cant reproduce the intercom issue, can you tell me more about what led to it?

**Comment by unit0016** [comment]
> cant reproduce the intercom issue, can you tell me more about what led to it?

TL;DR - Was playing AI and was setting up a spy network of Intercomms. Microphones on, speakers off, set to a frequency nobody would notice.

..Until someone in arrivals noticed. Then in robotics, then in command - no rhyme or reason, all of my intercomms were suddenly blaring their radio chatter..
..Despite all having their speakers turned off. Yes, I checked to confirm they weren't turned back on, and several audibly confirmed they still heard it despite it being off.

**Comment by Kylerace** [comment]
seems to happen if you turn off the speaker and THEN change frequencies. thank you for clarifying
